environment/index.js

```diff
@@ -56,7 +56,7 @@ RainbowConfig.declareOption(
 RainbowConfig.declareOption(
     'eth2near-client-contract-path',
     'The path to the Wasm file containing the Eth2NearClient contract.',
-    path.join(process.env.HOME, '.rainbowup/bridge/libs-rs/res/eth_client.wasm'),
+    path.join(__dirname, '../libs-rs/res/eth_client.wasm'),
```

Comment:
This actually doesn't help, because default option is override by `rainbow-bridge/config.json` and that one is a json we can't use __dirname there. But following this makes it work:
```
RainbowConfig.declareOption('bridge-src', 'Path to the rainbow-bridge source. Will use current repo if not provided.', path.join(__dirname, '../'));
```
And prepare.sh will `ln` current rainbow repo to `~/.rainbowup/bridge`. I'm thinking of erase `~/.rainbowup/bridge` complete, to make this less hacky